tests/rewrite.feature

```diff
@@ -115,42 +112,44 @@ Feature: Rewrites ExitTo Node Tags to Destination Way tags
             | a,c       | ab,bc,bc |
 
 
-    Scenario: Destination tag already present
+    Scenario: Tag already present
         Given the node map
             """
             a . . . b . . d .
                       ` . c .
             """
 
        And the nodes
-            | node | highway           | exit_to |
-            | b    | motorway_junction | ExitC   |
+            | node | highway           | exit_to | ref |
+            | b    | motorway_junction |       A |   2 |
 
         And the ways
-            | nodes | highway       | oneway | destination  |
-            | abd   | motorway      |        |              |
-            | bc    | motorway_link | yes    | DestinationC |
+            | nodes | highway       | oneway | destination  | junction:ref |
+            | abd   | motorway      |        |              |              |
+            | bc    | motorway_link | yes    |            B |            2 |
 
        When I route I should get
             | waypoints | route    |
             | a,c       | ab,bc,bc |
 
-    Scenario: Destination ref tag already present
+
+    Scenario: Ref tag already present
         Given the node map
             """
             a . . . b . . d .
                       ` . c .
             """
 
        And the nodes
-            | node | highway           | exit_to |
-            | b    | motorway_junction | ExitC   |
+            | node | highway           | exit_to | ref |
+            | b    | motorway_junction |       A |   2 |
 
         And the ways
-            | nodes | highway       | oneway | destination:ref |
-            | abd   | motorway      |        |                 |
-            | bc    | motorway_link | yes    | DestinationC    |
+            | nodes | highway       | oneway | destination:ref | junction:ref |
+            | abd   | motorway      |        |                 |              |
+            | bc    | motorway_link | yes    |               B |            3 |
```

Comment:
could you add a test for reversed oneways? From above I would expect them to not be handled correctly.